pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -32,6 +32,7 @@ const hasBeenActiveAnnotation = "HasBeenActive"
 
```

Comment:
`const hasBeenActiveAnnotation = "HasBeenActive"` <- this can be removed as well.